MvvmCross/Windows/Uwp/Views/MvxWindowsViewPresenter.cs

```diff
@@ -89,8 +192,72 @@ public override void Show(MvxViewModelRequest request)
             catch (Exception exception)
             {
                 MvxTrace.Trace("Error seen during navigation request to {0} - error {1}", request.ViewModelType.Name,
-                               exception.ToLongString());
+                    exception.ToLongString());
+            }
+        }
+
+        public MvxBasePresentationAttribute GetPresentationAttribute(Type viewModelType)
```

Comment:
For GetPresenationAttribute, CreatePresentationAttribute and GetOVerridePresentationAttribute we should follow what's being done for other platforms (eg https://github.com/MvvmCross/MvvmCross/blob/a78401df531d18b2f4bbf2120dd8a068d3c3b6c2/MvvmCross/Mac/Mac/Views/Presenters/MvxMacViewPresenter.cs)